.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,17 @@
-* [ ] Have you followed the guidelines in our
-      [Contributing document](https://dvc.org/doc/user-guide/contributing/core)?
+* [ ] ❗ Have you followed the guidelines in the
+      [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core)
+      list?
 
-* [ ] Does your PR affect documented changes or does it add new functionality
-      that should be documented? If yes, have you created a PR for
-      [dvc.org](https://github.com/iterative/dvc.org) documenting it or at
-      least opened an issue for it? If so, please add a link to it.
+* [ ] 📖 This PR does _not_ require [documentation](https://dvc.org/doc) updates
+      or modifications. If it _does_, have you created a PR for
+      [dvc.org](https://github.com/iterative/dvc.org) or at least opened an
+      issue for it? If so, please add a link to it and after that check this
+      box.
+
+* [ ] ❌ Have you checked DeepSource, CodeClimate, and other sanity checks
+      below? Their findings are considered recommendatory. We don't expect all
```

Comment:
"We consider their findings recommendatory and don't expect everything to be addresses. Please review them carefully and fix those that actually improve code or fix bugs."